core/src/main/java/org/opensearch/sql/executor/pagination/CanPaginateVisitor.java

```diff
@@ -68,14 +67,24 @@ public Boolean visitRelation(Relation node, Object context) {
     return Boolean.TRUE;
   }
 
-  private Boolean canPaginate(Node node, Object context) {
+  protected Boolean canPaginate(Node node, Object context) {
```

Comment:
why?

Reply:
I had a test which uses (overrides) it, but it become irrelevant after merging with #1500.